package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.12.1",
```

Comment:
These changes are irrelevant to the task (#15). Please, work move these changes to the corresponding different branch